Makefile

```diff
@@ -5,6 +5,9 @@ test: test-unit test-integration
 test-unit:
 	GO111MODULE=on go test -v ./...
 
+test-unit-cover:
+	GO111MODULE=on go test -v -coverprofile=coverage.txt -covermode=atomic ./...
```

Comment:
Add `-coverpkg` to track interpackage coverage. By default, coverage is only collected for the package under test.

Reply:
I wondered how to do that! Do you think we should though? I'm not sure we should encourage unit tests that test other packages indirectly...🤔

Reply:
I find that it gives way more realistic coverage. 

---

It is feature of go I wanted for a long long time.
go-ipfs still used Make rules I've wrote years ago to achieve the same: https://github.com/ipfs/go-ipfs/blob/9033ba9cadd48c15f848a15bbdccc21706124870/coverage/Rules.mk#L21-L25

Reply:
+1 in favor of adding `-coverpkg=all` - especially since package boundaries in the current code base aren't well aligned with where testing is happening.

Reply:
Okay! I'll add it, but we I think we should revisit when package boundaries are refactored to be clearer.